### PR TITLE
Replace {x} and {y} individually

### PR DESCRIPTION
**Pull request by bearfriend**
@m6jones

### CODE REVIEW

File: components/meter/meter-linear.js [hunk 1]
```diff
@@ -115,6 +115,8 @@ class MeterLinear extends LitElement  {
 		// Floor so you don't get 100% when you are at 99.5%
 		context = context.replace('{%}', `${Math.floor(value / max * 100)}%`);
 		context = context.replace('{x/y}', `${value}/${max}`);
```
Comment (m6jones) [inline components/meter/meter-linear.js]: Should we remove this?

Comment (bearfriend) [reply components/meter/meter-linear.js]: If no one else is using this currently, yeah.

**Comment by bearfriend** [comment]
@m6jones Yes, please.

**Comment by m6jones** [comment]
https://github.com/BrightspaceUI/core/pull/123

**Pull request closed without merging**